### PR TITLE
fix(GeneralLedger): totals are toFixed(2)

### PR DESCRIPTION
**Pull request by jniles**
This commit fixes the display of totals in the footer by forcing them to be toFixed(2).  It also updates the client JS to eslint standards.

generalledgerfootertotalsshouldbezero
_Fig 1: The footer should be toFixed(2)_

### CODE REVIEW

**Comment by jniles** [comment]
@mbayopanda, I discovered the sums in JavaScript are really inaccurate with Tshikajidata, so I switched them to SQL sums.  Could you give me a review?

File: client/src/modules/general-ledger/general-ledger.ctrl.js [hunk 1]
```diff
@@ -50,18 +46,18 @@ function GeneralLedgerController(
 
   vm.indentTitleSpace = 15;
 
-  tmpl =
-    '<div class="ui-grid-cell-contents">' +
-    '<span style="padding-left : {{row.entity.$$treeLevel * grid.appScope.indentTitleSpace}}px;"></span>' +
-    '{{grid.getCellValue(row, col)}}' +
-    '</span>';
+  const tmpl = `
```
Comment (mbayopanda) [inline client/src/modules/general-ledger/general-ledger.ctrl.js:49]: 👍

File: client/src/modules/general-ledger/general-ledger.ctrl.js [hunk 2]
```diff
@@ -50,18 +46,18 @@ function GeneralLedgerController(
...
+    <div class="ui-grid-cell-contents">
+      <span style="padding-left : {{row.entity.$$treeLevel * grid.appScope.indentTitleSpace}}px;"></span>
+      {{grid.getCellValue(row, col)}}
+    </span>`;
 
 
   function customAggregationFn(columnDefs, column) {
-    return (vm.aggregates[column.field] || 0);
+    return (vm.aggregates[column.field] || 0).toFixed(2);
```
Comment (mbayopanda) [inline client/src/modules/general-ledger/general-ledger.ctrl.js:57]: 👍

Comment (mbayopanda) [reply client/src/modules/general-ledger/general-ledger.ctrl.js:57]: ❓  The other thing is i just see zeros on the bottom of columns of the general ledger

**Review by mbayopanda — COMMENTED** [review]
I noticed zeros in bottom of columns of the general ledger without anyother value

File: client/src/modules/general-ledger/general-ledger.ctrl.js (hunk 2 above)
Comment (jniles) [reply client/src/modules/general-ledger/general-ledger.ctrl.js:57]: Yup!   All transactions should always sum to zero.  That means it is balanced 👍 

The bug I fixed was that it wasn't balanced and I couldn't figure out why .... but it seems like it was just JS math being horrible.

**Review by mbayopanda — APPROVED** [review]
LGTM

bors r+

**Pull request merged**